gnuradio-runtime/gnuradio-runtime.conf.in

```diff
@@ -13,10 +13,10 @@ max_messages = 8192
 #buffer_size = 32768
 
 [LOG]
-# Levels can be (case insensitive):
-#       DEBUG, INFO, WARN, TRACE, ERROR, ALERT, CRIT, FATAL, EMERG
+# levels, in ascending order of severity:
+# trace, debug, info, warning, error, critical, off
 log_level = info
-debug_level = emerg
+debug_level = critical
```

Comment:
This change in particular has caused a bit of trouble, because GNU Radio prior to version 3.10 raises an exception when it encounters an unknown log level, and `critical` is not in the list:

https://github.com/gnuradio/gnuradio/blob/fe1b3592776c26e349e9985d47d3178568dfe6ec/gnuradio-runtime/lib/logger.cc#L169-L196

Gqrx's AppImage releases are presently built on top of GNU Radio 3.8, and they fail to start if GNU Radio 3.10.7.0 or later is installed: https://github.com/gqrx-sdr/gqrx/issues/1316

I'll probably resolve this by updating the AppImage to use GNU Radio 3.10, but I wanted to note the problem here in case other users with multiple GNU Radio versions run into the same problem.